app/Http/Controllers/MemberController.php

```diff
@@ -12,63 +12,59 @@
 
 class MemberController extends Controller
 {
-    public function listMember(Request $request)
+    public function index(Request $request)
     {
-        $listMember= Member::all()->toArray();
+        $listMember = Member::all() ->toArray();
```

Comment:
Like "Member::all()->toArray()" it's better.

Reply:
I have fix